pkgs/top-level/aliases.nix

```diff
@@ -431,6 +431,7 @@ mapAliases {
   alsaTools = throw "'alsaTools' has been renamed to/replaced by 'alsa-tools'"; # Converted to throw 2024-10-17
   alsaUtils = throw "'alsaUtils' has been renamed to/replaced by 'alsa-utils'"; # Converted to throw 2024-10-17
   amazon-qldb-shell = throw "'amazon-qldb-shell' has been removed due to being unmaintained upstream"; # Added 2025-07-30
+  amdvlk = throw "'amdvlk' has been removed since it was deprecated by AMD. They suggest using RADV instead"; # Added 2025-09-20
```

Comment:
Maybe we can slightly improve the error message to make it clear that no action is required other than removing amdvlk (as radv/mesa works by default)?

Something like this?
> 'amdvlk' has been removed since it was deprecated by AMD. They suggest using RADV/mesa instead, which is enabled by default